velox/docs/develop/simple-vector-functions.rst

```diff
@@ -1,6 +1,6 @@
-=============================
-How to add a scalar function?
-=============================
+=======================================
+How to add a simple or vector function?
```

Comment:
I think "scalar" is a better term to use in the title.

Reply:
To clarify, keep the previous `How to add a scalar function?` or make this `How to add a scalar or vector function?`

Reply:
I'd keep the original "scalar function".